vcd/resource_vcd_nsxt_ipsec_vpn_tunnel.go

```diff
@@ -90,6 +108,12 @@ func resourceVcdNsxtIpSecVpnTunnel() *schema.Resource {
 				Required:    true,
 				Description: "Public IPv4 Address of the remote device terminating the VPN connection",
 			},
+			"remote_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "Custom remote ID. 'remote_ip_address' is used by default",
```

Comment:
Should it be "remote ID of the peer site"?

Reply:
Yes, this is a valid comment. changed.